client/src/Components/FormInput.js

```diff
@@ -53,7 +53,7 @@ function FormInput({
           value={value || ''}
           onKeyDown={handleKeyDown}
           onBlur={handleOnBlur}
-          valueAsNumber={type === 'number'}
+          valueasnumber={(type === 'number').toString()}
```

Comment:
This isn't a settable property.  It's just for getting the current value as a number instead of a string.

Reply:
initially when I had valueAsNumber, I was getting a warning in the console saying it wasn't a valid react prop, and that I needed to use valueasnumber. After the change, I got a different warning, stating the value had to be a string of true or a string of false.